source/docs/hardware/hardware-safety/placeholder.rst

```diff
@@ -0,0 +1,44 @@
+Placeholder for Safety Writeup
+==============================
+
+.. Note:: From a Chief Delphi `thread reply <https://www.chiefdelphi.com/t/severe-100-packet-loss/428031/139?u=peter_johnson/>`_ by WPILib Developer Peter Johnson.
```

Comment:
This should be in the PR description/commit message, but doesn't need to be in the article